[A cellular entity retaining only its replicative core: Hidden archaeal lineage with an ultra-reduced genome](https://www.biorxiv.org/content/10.1101/2025.05.02.651781v1) #har25 (2025) #preprint 

"Here, we report the discovery of a novel archaeon, *Candidatus Sukunaarchaeum mirabile*, ... This archaeon possesses an extraordinarily small genome (238 kbp) and represents a deeply branching, previously unknown lineage distinct from established archaeal phyla. Its profoundly stripped-down genome, largely devoid of metabolic functions, retains **primarily the core machinery for self-replication** (DNA replication, transcription, translation)." p. 3

- 238 kbp - "less than half the size of the smallest previously known archaeal genome" (Abstract)

"The Sukunaarchaeum genome displays an extreme functional bias towards genetic information processing ... GIP-related [genetic information processing] genes constitute 52.4% of all protein-coding genes (Figure 4a left, ...) and 72.6% of those with functional predictions. Conversely, proteins responsible for metabolism were notably absent (Figure 4a middle, ...), indicating a striking bias in the function of encoded proteins." p. 6

![[har25 fig 4a - genome size scatterplots.png]]
**Figure 4a** The red dot in each scatterplot is Sukunaarchaeum compared to other archaea/bacteria

"This distinct functional profile suggests that Sukunaarchaeum may have been exposed to extreme genome reduction pressures comparable to those on the bacterial endosymbionts, but that its gene retention was shaped by different selective constraints." p. 6

"The near-complete absence of recognizable metabolic pathways in Sukunaarchaeum precludes any obvious metabolic contribution to its host and strongly suggests an obligate, potentially parasitic, dependence in which it unilaterally exploits host resources." p. 7

"This extreme functional specialization is underscored by the retention of most core protein components essential for transcription, translation, and DNA replication ... despite possessing only 189 protein-coding genes." p. 7

"Only five proteins appear clearly unrelated to these core processes: four are transporter subunits (three ABC transporter, one Sec transporter), presumably involved in acquiring essential molecules from the host environment. The fifth shows weak homology to phosphoglycolate phosphatase ...; however, given the absence of canonical pathways producing its substrate (2-phosphoglycolate) and the low sequence similarity, its true function remains enigmatic and may involve a different substrate entirely." p. 7

"Despite its highly reduced genome, Sukunaarchaeum encodes several unusually large proteins, a feature uncommon in organisms with such small genomes. ... The retention of these large proteins, despite the strong evolutionary pressure for genome reduction expected in Sukunaarchaeum, suggests their crucial roles in the lifestyle of Sukunaarchaeum." p. 8

"Among the 13 Sukunaarchaeum proteins exceeding 1,000 [amino acids], homology searches revealed the functions of the five smallest proteins: RNA polymerase subunits, DNA polymerase, topoisomerase, and LonB protease. However, the **remaining eight larger proteins showed no apparent sequence similarity to any known proteins**, hindering functional inference. Notably, the coding regions for these eight large proteins comprise 25% of the Sukunaarchaeum genome (59,946 out of 238,034 bp; Figure 4c) p. 8

![[har25 fig 4c - gene count and length proportions.png]]
**Figure 4c** Proportions of *number* of genes (left), and genomic sequence *length* (right)

"These results strongly suggest that the large proteins of unknown function in Sukunaarchaeum are localized and function at the cell membrane. ... Interestingly, the genes encoding four of the large proteins ... are not only located on the same strand within the genome but are also adjacent to each other, suggesting a functional relationship." p. 9

![[har25 stbl - 4 large genes are consecutive.png]]

[Supplementary Table](https://www.biorxiv.org/content/biorxiv/early/2025/05/02/2025.05.02.651781/DC13/embed/media-13.xlsx?download=true) (Tab S2) Four of the large proteins are adjacent on the genome

"... mapping the transmembrane (TM) helix regions predicted by several software onto the predicted three-dimensional structure of one of the large proteins (SKNCREG01_0060) revealed that the TM regions appear to align on a single plane" p. 9

![[har25 fig s11 - transmembrane helix regions.png]]

**Figure s11b** Predicted 3D structure of one of the large proteins. Blue helices are predicted to lie on the host membrane

"These results further support our hypothesis that the large proteins are cellular membrane proteins." p. 9
